### PR TITLE
Feature - Minimum quantity and quantity increment

### LINKED ISSUE #1366: Minimum quantity and quantity increment

### PR DESCRIPTION
**Pull request by alecritson**
Closes #1366

### CODE REVIEW

File: packages/admin/resources/lang/en/product.php [hunk 1]
```diff
@@ -92,6 +92,18 @@
                         'backorder' => 'Backorder Only',
                     ],
                 ],
+                'unit_quantity' => [
+                    'label' => 'Unit Quantity',
+                    'helper_text' => 'How many individual items make up 1 unit.',
+                ],
+                'min_quantity' => [
+                    'label' => 'Minimum Quantity',
+                    'helper_text' => 'The fewest number of items of a product variant that can be purchased at the same time.',
```
Comment (glennjacobs) [inline packages/admin/resources/lang/en/product.php]: Doesn't read that great. Can Grammarly or ChatGPT help out?

File: packages/admin/resources/lang/en/product.php [hunk 2]
```diff
@@ -92,6 +92,18 @@
...
+                ],
+                'quantity_increment' => [
+                    'label' => 'Quantity Increment',
+                    'helper_text' => 'The number of items by which a product variant can be purchased.',
```
Comment (glennjacobs) [inline packages/admin/resources/lang/en/product.php]: Not even sure this explains it properly. Maybe something like `The product variant must be purchased in multiples of this quantity`.

File: packages/admin/resources/lang/en/product.php (hunk 1 above)
Comment (alecritson) [reply packages/admin/resources/lang/en/product.php]: Grammarly didn't come up with anything when using the IDE and their website requires premium to suggest anything. I don't know of any decent ChatGPT services, but I'm happy to take suggestions.

File: packages/admin/resources/lang/en/product.php (hunk 2 above)
Comment (alecritson) [reply packages/admin/resources/lang/en/product.php]: Updated to your suggestion.

File: packages/admin/resources/lang/en/product.php (hunk 1 above)
Comment (glennjacobs) [reply packages/admin/resources/lang/en/product.php]: `The minimum quantity of a product variant that can be bought in a single purchase.`

File: packages/admin/resources/lang/en/product.php (hunk 2 above)
Comment (glennjacobs) [reply packages/admin/resources/lang/en/product.php]: `The product variant should be bought in increments of this specified quantity.`

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**